playbooks/common/openshift-master/config.yml

```diff
@@ -334,8 +334,8 @@
   serial: 1
   vars:
     sync_tmpdir: "{{ hostvars.localhost.g_master_mktemp.stdout }}"
-    openshift_master_ha: "{{ groups.oo_masters_to_config | length > 1 }}"
-    openshift_master_count: "{{ groups.oo_masters_to_config | length }}"
+    openshift_master_ha: "{{ groups.oo_masters | length > 1 }}"
```

Comment:
Should we persist the value of openshift_master_ha (and not setting it to false here) to avoid changing the master configuration too substantially if we run against only a single master?